catalyst_voices_packages/catalyst_cardano_serialization/lib/src/types.dart

```diff
@@ -75,7 +83,7 @@ extension type const SlotBigNum(int value) {
 }
 
 /// Represents the balance of the wallet in terms of [Coin].
-final class Balance extends Equatable {
+final class Balance extends CborEncodable {
```

Comment:
`Balance` should extend `Equatable` and implement `CborEncodable`, otherwise hashCode and equals don't work

Reply:
Sorry, I's pushed now and it should be solved.